src/components/Modal/MemberProfile.tsx

```diff
@@ -0,0 +1,45 @@
+import Close from '../../../public/images/Close.svg';
+import userImg from '../../../public/images/userImgNull.svg';
+
+interface MemberProfileProps {
+  // wrong?: () => void;
+  wrong?: any;
+  getMemberId?: any;
+}
+
+function MemberProfile({ wrong, getMemberId }: MemberProfileProps) {
+  const TextStyles = 'text-[#D8D8D8] text-[1.4rem] mt-5';
+  const { grade, name, profileImg, role } = getMemberId;
+
+  const handleMemberEdit = () => {
+    console.log('Edit Button Click!');
+  };
+  return (
+    <div className="fixed right-4 top-0 z-10 flex size-full  items-center justify-center bg-black bg-opacity-5">
+      <div className="h-[48.5rem] w-[48.2rem] border-[0.1rem] border-[#DCDCDC] bg-[#FFF] pl-8 pr-12 pt-[1.8rem]">
+        <div className="mb-[4.5rem] flex items-center justify-between text-[1.6rem]">
+          프로필
+          <button onClick={wrong}>
+            <img src={Close} alt="CloseImg" />
+          </button>
+        </div>
+        <div className="flex flex-col items-center">
+          {profileImg !== 'null' ? <div>해당 이미지</div> : <img src={userImg} alt="userBigImg" />}
```

Comment:
```suggestion
          {profileImg ? <div>해당 이미지</div> : <img src={userImg} alt="userBigImg" />}
```
프로필 이미지값이 있으면 해당 이미지, 없으면 기본 이미지 보여주는 코드라면 이 부분도 이렇게 바꿀 수 있겠네요

Reply:
오오 그렇네요 생각해보니

Reply:
근데 지금은 sting으로 넘겨줘서 저렇게 하기엔 무리가 있을것 같아요 실제 데이터 받을때 수정 하겠습니다 ^_^